src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -285,12 +287,9 @@ function ReactDevtoolsPanel({
     onShutdown
   );
 
-  const { value: enableDarkMode } = useFeature("darkMode");
-  let theme = enableDarkMode ? "dark" : "light";
-
   return (
     <ReactDevTools
-      browserTheme={theme}
+      browserTheme={enableDarkMode ? "dark" : "light"}
```

Comment:
Lol that's a whole lot simpler, eh

Reply:
small detail. the big one was fixing the lint issue